packages/sdks/snippets/react/src/components/CustomColumns.tsx

```diff
@@ -0,0 +1,35 @@
+import { Blocks, BuilderBlock } from "@builder.io/sdk-react";
+import { Key } from "react";
+
+interface CustomColumnProps 
+{ 
+  inside: { 
+    blocks: BuilderBlock[] | undefined; 
+  }[];
```

Comment:
i find the `inside` here a bit unusual. Can we remove it and have it be `props.blocks` directly?